src/components/checkboxes/Checkboxes.tsx

```diff
@@ -1,18 +1,27 @@
 import React, { HTMLProps, PureComponent, SyntheticEvent } from 'react';
 import classNames from 'classnames';
+import LabelBlock from '../utils/LabelBlock';
+import { HintProps } from '../hint/Hint';
+import { generateRandomName } from '../../util/RandomName';
 import FormContext from '../form/FormContext';
-import ErrorMessage from '../error-message';
 import CheckboxContext from './CheckboxContext';
 import Box, { BoxProps } from './Box';
+import { LabelProps } from '../label/Label';
+import { ErrorMessageProps } from '../error-message/ErrorMessage';
 
 interface CheckboxesEvent extends SyntheticEvent<HTMLInputElement> {
   target: HTMLInputElement;
 }
 
 interface CheckboxesProps extends HTMLProps<HTMLDivElement> {
-  error?: string | boolean;
   idPrefix?: string;
   onChange?: (e: CheckboxesEvent) => any;
+  label?: string;
```

Comment:
Just wondering, since these types are now used across 6 different input components now, is it worth having an interface that contains label, hint, and error props and extending from that?